src/components/NewCard/docs/NewCardExample.tsx

```diff
@@ -0,0 +1,140 @@
+                I have 200px min-width, and the media has 300px min-width, so I
+                break when the total width is &lt; 500px
+              </Text>
+            </div>
+          </NewCard.Container>
+          <NewCard.Container className={styles.media} minWidth={300}>
+            {media}
+          </NewCard.Container>
+        </NewCard>
+
+        <br />
+        <br />
+
+        <NewCard className={styles.card + ' ' + styles.example2} stacked>
+          <NewCard.Container className={styles.media}>
+            {media}
+          </NewCard.Container>
+          <NewCard.Container className={styles.info}>
+            <div>
+              <Text typography={TYPOGRAPHY.smallTitle}>Stacked</Text>
+              <br />
+              <Text typography={TYPOGRAPHY.runningText}>
+                My width can grow and shrink
+              </Text>
+            </div>
+          </NewCard.Container>
+        </NewCard>
+
+        <br />
+        <br />
+
+        <NewCard className={styles.card + ' ' + styles.example3}>
+          <NewCard.Container className={styles.media}>
+            {media}
+          </NewCard.Container>
+          <NewCard.Container className={styles.info}>
+            <div>
+              <Text typography={TYPOGRAPHY.smallTitle}>Strip card</Text>
+              <br />
+              <Text typography={TYPOGRAPHY.runningText}>Yes we can!</Text>
+            </div>
+          </NewCard.Container>
+        </NewCard>
+
+        <br />
+        <br />
+
+        <NewCard className={styles.card + ' ' + styles.dugma}>
+          <NewCard.Container className={styles.media} minWidth={300}>
+            {media}
+          </NewCard.Container>
+          <NewCard.Container className={styles.info} minWidth={200}>
+            <div>
+              <Text typography={TYPOGRAPHY.smallTitle}>
+                Side By Side - Only root container has border
+              </Text>
+              <br />
+              <Text typography={TYPOGRAPHY.runningText}>
+                I have 200px min-width, and the media has 300px min-width, so I
+                break when the total width is &lt; 500px
+              </Text>
+            </div>
+          </NewCard.Container>
+        </NewCard>
+
+        <br />
+        <br />
+
+        <NewCard className={styles.card + ' ' + styles.overlapping}>
+          <NewCard.Container className={styles.media} minWidth={300}>
+            {image}
+          </NewCard.Container>
+          <NewCard.Container className={styles.info} minWidth={200}>
+            <div className={styles.overlapContent}>
+              <Text typography={TYPOGRAPHY.smallTitle}>Overlapping</Text>
+              <br />
+              <Text typography={TYPOGRAPHY.runningText}>
+                I have 200px min-width, and the media has 300px min-width, so I
+                break when the total width is &lt; 500px
+              </Text>
+            </div>
+          </NewCard.Container>
+        </NewCard>
+      </div>
```

Comment:
So the designers asked me to remove this example, since they don't want people to use this type of layout.